### PR TITLE
ZCS-2423: Universal UI- Quick Add Contact Dialog

### PR DESCRIPTION
**Pull request by kalpeshhpatel**
Updated look and feel of Quick Add Contact dialog

https://jira.corp.synacor.com/browse/ZCS-2423

### CODE REVIEW

File: WebRoot/skins/_base/base4/skin.properties [hunk 1]
```diff
@@ -1491,6 +1491,12 @@ DLPropertiesViewRowLabel    = @GroupEditViewFontSize@; vertical-align:top; paddi
 DLPropertiesViewRowValue    = padding: 0 0 @SkinWrapperPadding@ 0; @GroupEditViewFontSize@;
 DLPropertiesViewInput       = width:45em; margin:0;
 
+# Quick Add Dialog
+ZmQuickAddDialog            = 
+ZmQuickAddDialogTable       = border-collapse:separate; border-spacing:0 @SkinViewInnerSpacing@;
+ZmQuickAddDialogFieldLabel  = padding-right:@SkinWrapperPadding@;
+ZmQuickAddDialogInput       = @FixBoxModel@; @FullWidth@; @PanelBorder@; @MediumRoundCorners@;
```
Comment (tarangkhandelwal) [inline WebRoot/skins/_base/base4/skin.properties]: should we make this generic ? Might need similar inputs in other dialogs too?

File: WebRoot/css/zm.css [hunk 2]
```diff
@@ -7139,6 +7139,27 @@ a.ZmLinkDisabled {
 	@ContactListPaginationLinkIcon@
 }
 
+/* Quick add contact dialog*/
+.ZmContactQuickAddDialog {
+	@ZmQuickAddDialog@
```
Comment (tarangkhandelwal) [inline WebRoot/css/zm.css]: ZmQuickAddContactDialog maybe?

File: WebRoot/templates/abook/Contacts.template [hunk 3]
```diff
@@ -1142,19 +1142,19 @@
 <template id='abook.Contacts#QuickAddPrompt'>
 	<table role="presentation" class='ZPropertySheet' cellspacing='6'>
 		<tr>
-			<td class='ZmFieldLabelRight'><$=ZmMsg.firstNameLabel$></td>
+			<td class='ZmFieldLabel'><$=ZmMsg.firstNameLabel$></td>
 			<td><div id='${id}_FIRST_NAME'></div></td>
 		</tr>
 		<tr>
-			<td class='ZmFieldLabelRight'><$=ZmMsg.lastNameLabel$> </td>
+			<td class='ZmFieldLabel'><$=ZmMsg.lastNameLabel$> </td>
```
Comment (tarangkhandelwal) [inline WebRoot/templates/abook/Contacts.template]: ZmFieldLabelLeft ? or is it ok as well? I am not sure

Comment (kalpeshhpatel) [inline WebRoot/templates/abook/Contacts.template]: Well, `ZmField` also woks, as by default it is left aligned.

Thanks

File: WebRoot/css/zm.css (hunk 2 above)
Comment (kalpeshhpatel) [inline WebRoot/css/zm.css]: Will work, but I have choose that name to reduce number of chars. As subsequent property names usually uses `ZmQuickAddDialog` as prefix. 

It just my preference, however I am open to change it if you feels so. 

Let me know. 

Thanks

File: WebRoot/skins/_base/base4/skin.properties (hunk 1 above)
Comment (kalpeshhpatel) [inline WebRoot/skins/_base/base4/skin.properties]: Thanks for bringing it up. 
I'll do it.

File: WebRoot/templates/abook/Contacts.template (hunk 3 above)
Comment (kalpeshhpatel) [inline WebRoot/templates/abook/Contacts.template]: Updated here: https://github.com/Zimbra/zm-web-client/pull/116/commits/eb03e8cd7e55dfaa5a6af0121d43726fa819c775#diff-ca538f2b6603f14c9fb50d9afdbdb7cbR1145

File: WebRoot/skins/_base/base4/skin.properties (hunk 1 above)
Comment (kalpeshhpatel) [inline WebRoot/skins/_base/base4/skin.properties]: Updated here: https://github.com/Zimbra/zm-web-client/pull/116/commits/eb03e8cd7e55dfaa5a6af0121d43726fa819c775#diff-16ae02866e4f63813de9a7825a1057e3R480

Comment (tarangkhandelwal) [inline WebRoot/skins/_base/base4/skin.properties]: looks good.

File: WebRoot/css/zm.css (hunk 2 above)
Comment (tarangkhandelwal) [inline WebRoot/css/zm.css]: For me the property name seem something generic for anything that's like QuickAdd. Contact/Appointment etc ...That's why I was looking to add something specific.

Comment (kalpeshhpatel) [inline WebRoot/css/zm.css]: Hmm...makes sense. I have updated changes here: https://github.com/Zimbra/zm-web-client/pull/116/commits/f4d5f34cda9f3da808b2dfd26028e9ec4bc29acc

Thanks

**Pull request merged**